nikolai4D/frontEndFramework
# Uc 91 add add project modal to project view

UC-91-add-add-project-modal-to-project-view


## Description & motivation

Creating modal when clicking on button on projects view.
Modal contains new created organism and molecules.


## How to test

Add #UC-91-add-add-project-modal-to-project-view at the end of "nd_frontend" in package.json in UrbanCloud
Run and start the project.
Go to view /projects
Click on button above cards to see modal. 



## New tickets to be added





---
